ipn/ipnlocal/profiles.go

```diff
@@ -443,12 +448,20 @@ func (pm *profileManager) writeKnownProfiles() error {
 	return pm.WriteState(ipn.KnownProfilesStateKey, b)
 }
 
+func (pm *profileManager) updateHealth() {
```

Comment:
Sorry, @maisem, but this is the only way I could find to update all the bazillion paths that changed prefs in LocalBackend :(

Reply:
seems fine